targets/csharp-unityv2/source/Shared/Internal/PlayFabVersion.cs

```diff
@@ -0,0 +1,14 @@
+
+namespace PlayFab.Internal
+{
+    public class PlayFabVersion
+    {
+        public static string SdkRevision = "0.23.160414";
```

Comment:
All sdks have been converted to use SdkVersion, including V1.
rename SdkRevision to SdkVersion.